modules/pantheon/pantheon_apachesolr/pantheon_apachesolr.module

```diff
@@ -157,6 +157,17 @@ function pantheon_apachesolr_curlopts() {
   );
 }
 
+function pantheon_apachesolr_certs_dir() {
+  if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+    return $_SERVER['HOME'] . '/certs';
+  }
+  return '../certs';
```

Comment:
It seems unlikely that we would want to support a local development environment for Pantheon ApacheSolr; if we did, though, we could look at `git rev-parse --show-toplevel` here to find the root of the git repository, and perhaps look for a `certs` dir one level above that.

It is probably sufficient to support only the Pantheon environment here, though.